src/pages/preview/[...slug].tsx

```diff
@@ -36,20 +41,14 @@ const ContentPage: NextPage<ContentPageProps> = ({
             contentItem={contentItem}
             tertiaryNavItems={tertiaryNavItems}
             relatedContent={relatedContent}
+            previewMode={previewMode}
         />
     );
 };
 
 export default ContentPage;
 
 export const getServerSideProps = async (context: any) => {
```

Comment:
Should we be checking here for `context.preview`? I'm following along here https://nextjs.org/docs/advanced-features/preview-mode#step-2-update-getstaticprops. I may be misunderstanding, but it seems like anyone could pass in the slug of an unpublished article and see it by going to `developer/preview/<some_draft_slug>`?

Reply:
@mcolella14 true.Context.preview is always coming as true whether it's coming from redirect or url is directly from preview url, same with Context.previewData. I'm exploring how to have a route protected in next.js, but don't have luck yet.

Reply:
Have you tried going directly to the preview route from an incognito window? I think the preview mode sets a cookie on your browser, so it would make sense for `context.preview` to be `true` if you went to the preview page route after hitting the preview api route a while ago. If it's still true with an incognito window, then I have no idea lol.